.github/workflows/acceptance-tests.yml

```diff
@@ -31,13 +31,16 @@ jobs:
       serverless: ${{ steps.filter.outputs.serverless }}
       network: ${{ steps.filter.outputs.network }}
       config: ${{ steps.filter.outputs.config }}
+      assume_role: ${{ steps.filter.outputs.config }}
     steps:
     - uses: actions/checkout@v4
       if: ${{ github.event_name == 'push' ||  github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || inputs.parent-event-name == 'release' }}
     - uses: dorny/paths-filter@v2
       id: filter
       with:
         filters: |
+          assume_role:
+            - 'mongodbatlas/**provider**.go'
```

Comment:
this seems very broad and it's also matched e.g. in config, what list of files do we want to match here?

Reply:
I thought of making it broad because I'd thought it should run for all PR changes since it's an authentication type of test. Happy to change here later if we see it's not worth it

Reply:
in that case we may want to have it in the general "config" group instead of a separated asssume-role group.

Reply:
let's see how it goes, I'll keep this comment into account to improve things. Let us just see how it works.